wordpress-mobile/WordPress-iOS
# Issues/4094 reader menu sites

Refs #4094
This is pr number 3 of the menu refresh.  This pr introduces following management for sites to the new menu. It includes the following: 
- Ports the old following vc to swift, *mostly* intact.
- Switches from the ReaderSite entity to the ReaderSiteTopic entity for core data concerns. The ReaderSite entity should be considered deprecated from this point forward and will be removed via a clean up PR following this feature going live. 
- Adds a new field to ReaderSiteTopic to fix an issue unfollowing feeds. Fixes #5422. 
- Adds the ability to preview / browse posts from sites you follow from the new following controller. 

To test:
- On the reader menu tap the "manage" button to launch the site management
- Try following sites and feeds by typing their urls in the search bar. 
- Test the swipe to delete gesture and unfollowing sites and feeds.
- Test browsing / previewing posts from the sites/feeds you follow
- Try unfollowing a site/feed you're previewing and confirm that it does not show up when you return to the list.
- Confirm the layout is correct on the iPad.

Needs review: @frosty would you take a peek at this one?
